pkg/virt-operator/application.go

```diff
@@ -159,10 +163,6 @@ func Execute() {
 		PodDisruptionBudgetCache:      app.informerFactory.OperatorPodDisruptionBudget().GetStore(),
 	}
 
-	onOpenShift, err := util.IsOnOpenshift(app.clientSet)
-	if err != nil {
-		golog.Fatalf("Error determining cluster type: %v", err)
-	}
 	if onOpenShift {
 		log.Log.Info("we are on openshift")
 		app.informers.SCC = app.informerFactory.OperatorSCC()
```

Comment:
app.informerFactory.OperatorSCC needs to change to only watch SCC objects with our operator label.

Reply:
In the current implementation, if I change that then operator won't add virt-handler to privileged SCC (we observe only SCC with operator label, privileged SCC doesn't have required label).

Hm, since I added SCC component, I think it's worth to create an additional `virt-privileged` SCC for virt-handler (and components which need that) and stop modifying the generic `privileged`, it would simplify the implementation (we could drop `addOrRemoveSCCUsers` func) and we would have full control over virt-handler permissions.

In my opinion, it's a good solution, what do you think?

Reply:
To give you an overview, I pushed that change quickly, https://github.com/kubevirt/kubevirt/pull/2545/files#diff-2c9edb5dc9218f571926c89df6f12435R45 and we got rid of https://github.com/kubevirt/kubevirt/pull/2545/files#diff-9a9763888082c586cc68fce7c54befd2L46

Reply:
yep, we don't want to modify the `privileged` SCC object anymore.